appIconIndicators.js

```diff
@@ -794,7 +794,19 @@ const UnityIndicator = new Lang.Class({
             return;
         }
 
-        this._progressOverlayArea = new St.DrawingArea({x_expand: true, y_expand: true});
+        this._progressOverlayArea = new St.DrawingArea({name: 'dashtodockContainer', x_expand: true, y_expand: true});
```

Comment:
There is no need for this to have 'dashtodockcontainer' as id/name. The parent (one of the dock container actor) has this id, and it should not be duplicated (although the shell probably won't complain). Instead, the style would be defined as `` #dashtodockContainer .progress-bar`` (added space), and it should work.

Reply:
Ok, I didn't know this and I wasn't able to make it work without the name

Reply:
It looks like that removing the `id/name` does not work. I tried first as you said, removing the name and adding the space between object and class in the style, but then shell complains it couldn't find the color property.

Reply:
mmm, sorry, I was thinking ahead. At the moment you add the element to the uiGroup when you retrieve the style, therefore it is not a children of th dock container at that point. However, if  as I was suggesting you permanently apply the class to the StDrawingArea and retrieve the style when that is added to the icon (which is a children of the dashtodockContainer element already), it should work.

Reply:
Oh, I see now, thanks for your patience